spec/system/planning_applications/constraints_spec.rb

```diff
@@ -32,7 +32,6 @@
       expect(page).to have_content("Listed building")
       expect(page).to have_content("Broads")
       expect(page).to have_content("Flood zone 3")
-      expect(page).to have_no_content("Conservation Area")
```

Comment:
note: this is because the audit log now has the text "Conservation Area" being displayed as one of the things removed, and the info is displayed on the same page although in a different accordion.